src/routers/GetRouteRouter.js

```diff
@@ -91,6 +92,37 @@ class GetRouteRouter extends AppDevRouter {
       }
     }
 
+     const stopSummary = [];
```

Comment:
Overall comment: these different string should be constants.  "walk" should be a constant, "place" should be a constant, etc.  If we need to change these every, we'd have to go back through the code and change the use of every one of these plain-text strings.